src/main/java/attendance/domain/PenaltyCrew.java

```diff
@@ -0,0 +1,59 @@
+package attendance.domain;
+
+import java.util.Objects;
+
+public class PenaltyCrew implements Comparable<PenaltyCrew> {
+
+    private final String name;
+    private final int absenceCount;
+    private final int lateCount;
+    private final Integer point;
+    private final AttendancePenalty attendanceStatus;
+
+    public PenaltyCrew(String name, int absenceCount, int lateCount) {
+        this.lateCount = lateCount;
+        this.absenceCount = absenceCount;
+        this.name = name;
+        this.point = absenceCount * 3 + lateCount;
+        this.attendanceStatus = AttendancePenalty.find(absenceCount, lateCount);
```

Comment:
point 와 attenanceStatus 는 `absenceCount` 와 `lateCount` 를 사용해서 나중에 계산하는것도 가능해 보여요
생성자에서 필드로 초기화 해두신 이유가 궁금해요 😃

Reply:
요구조건에 따라 출석 경고별 정렬, 지각을 결석으로 간주후 내림차순 정렬을 위해 생성자에서 초기화 하여 compareTo() 메서드에서 참조하기 위해 이렇게 했습니다. 만약 나중에 계산한다면, compareTo() 메서드에서 값을 직접 계산해서 비교로직을 짜는 방법을 말씀하시는 걸까요? 그럼 compareTo() 메서드가 복잡해지지 않을까요? 아니면 다른 방법이 있으신지 궁금합니다.

Reply:
compareTo 쪽을 생각못했어요 😅
좋은 의견 주셔서 감사해요~
PenaltyCrew는 불변 객체이고 point 와 attendanceStatus 또한 불변이니
생성자에서 초기화하는것도 좋다고 생각합니다!

+) point 계산하는 부분은 메서드로 추출해보는 것도 좋다고 생각해요!

```suggestion
        this.point = this.calculatePoint(absenceCount, lateCount);
        this.attendanceStatus = AttendancePenalty.find(absenceCount, lateCount);
```

Reply:
생성자에서 직접 계산하는 것보다, 간단한 로직이라도 메서드로 분리하여 어떤 과정을 통해서 인스턴스 변수가 할당되는지 의미를 전달하는 것이 더 가독성 좋은 방법인것 같습니다. 앞으로 생성자를 사용할때, 값을 계산하거나 가공하는 과정이 있다면 메서드로 분리하는 것을 고려하겠습니다. point 변수명 보다 지각 횟수와 결석 횟수의 가중치를 이용하여 총 횟수를 계산한다는 의미가 적절하다고 생각하여 totalCount로 변수명을 변경했습니다.